### DISCUSSION TITLE
Running two instances of the pipeline

### DISCUSSION THREAD

**Discussion by jennysjaarda** [ropensci/targets#573] [Help]
I was just wondering, is it possible to run two instances of the pipeline that don't interact with each other? I run on an HPC cluster that has  a number of partitions and would like to have one section (sectionA) of my pipeline running on one partition while running another section (sectionB) on another (using slurm). SectionA has a very longtime runtime, and would like to progress in the rest of the pipeline while it is running if possible. I have tried and am not getting any errors, but as I build and add more targets onto SectionB, targets that have been run since SectionA started seem to be re-running, so I'm wondering if it's causing some problems. Any advice? 
Thanks!

**Comment by wlandau** [comment]
I would advise against this if both instances of the pipeline share the same data store, even if the targets being run in each instance are different. You might get lucky, but in the long run you might also cause an instance of https://github.com/ropensci/targets/issues/368 if two processes are writing to the same `_targets/meta/meta` metadata file.
